ecleankernel/layout/blspec.py

```diff
@@ -149,6 +146,15 @@ def find_kernels(self,
         for mkv, fobjs in module_dict.items():
             if any(mkv == k.real_kv for k in kernels.values()):
                 continue
-            kernels.setdefault(mkv, Kernel(mkv)).all_files.extend(fobjs)
+            # If this mkv already has an entry in kernels, use
```

Comment:
Good point, Fixed!

I initially used only `bls`, `uki`, `other` as layout identifiers since these are the three defined in the `kernel-install` manual. But I guess there is no reason why we couldn't define some new ones to be more descriptive.